app/controllers/choices_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ChoicesController < ApplicationController
+  def new
+    @question = Question.find(params[:question_id])
+    @choice = Choice.new
+  end
+
+  def create
+    @question = Question.find(params[:question_id])
+    @choice = Choice.new(choice_params)
+    @question.choice = @choice
+    if @choice.save
+      flash[:notice] = 'Answer choice successfully added'
+      redirect_to choices_path
+    else
+      flash[:error] = @question.errors.full_messages.join(' ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.